pkg/clusters/clusters.go

```diff
@@ -29,16 +34,16 @@ func GetClusters(r ClustersRetriever, w io.Writer) error {
 	}
 
 	if len(cs) > 0 {
-		fmt.Fprintf(w, "NAME\tSTATUS\n")
+		fmt.Fprintf(w, "NAME\tSTATUS\tSTATUS_MESSAGE\n")
 
 		for _, c := range cs {
-			if c.PullRequestType == "create" {
+			if c.PullRequest.Type == "create" {
 				c.Status = "Creation PR"
-			} else if c.PullRequestType == "delete" {
+			} else if c.PullRequest.Type == "delete" {
 				c.Status = "Deletion PR"
 			}
 
-			fmt.Fprintf(w, "%s\t%s", c.Name, c.Status)
+			fmt.Fprintf(w, "%s\t%s\t%s", c.Name, c.Status, c.PullRequest.Url)
```

Comment:
🙌 , maybe only if c.status is creation/deletion PR. e.g. I guess leave it blank for "clusterFound" etc.

Reply:
Sure! 👌 Thank you Simon 🙏